src/main/resources/quilt.mod.json

```diff
@@ -152,7 +152,6 @@
 			"quilt_status_effect",
 			"quilt_surface_rule",
 			"quilt_tags",
-			"quilt_testing",
```

Comment:
please keep it in; we do intentionally include quilt testing api on prod

Reply:
What are we using it for?

Reply:
the testing api module has some parameters that allow for it to be enabled on production; an example of it being used is me when i test QFAPI on prod :p (separate testmod jar is still required for the tests)